src/main/proto/widget.proto

```diff
@@ -0,0 +1,11 @@
+syntax = "proto3";
+
+package smartthings;
+
+option java_package = "smartthings.ratpack.protobuf";
+option java_outer_classname = "WidgetProtos";
+
+message Widget {
```

Comment:
If this is a test fixture, should it be part of src/main?  If this can't be compiled and utilized from tests while under src/test; maybe a sub project for test fixtures would be a good option.